src/pages/index.tsx

```diff
@@ -201,7 +212,18 @@ class IndexPage extends Component<IndexPageProps, IndexPageState> {
 						renderTableRow={crew => this.renderTableRow(crew)}
 						filterRow={(crew, filter) => this._filterCrew(crew, filter)}
 						config={tableConfig}
-					/>
+                        searchExt = { 
+                            <span style={{ paddingLeft: '2em' }}>
+                                <Dropdown inline
+                                          options={searchTypeOptions}
+                                          value={this.state.searchType}
+                                          onChange={(event, {value}) => 
+                                            this.setState({ searchType: value as number })
```

Comment:
Why is this cast to a number?

Reply:
Both are legacy code that I forgot to take out. The this.extSearch check was because it was originaly a function which broke other pages but now doesn't need a check. The cast was because I originally use an array of lambdas which I then converted to a dictionary but forgot to remove the cast. I'm suprised that it didn't break anything.

Reply:
In this case it could only break TypeScript, as it's not actually a type conversion at runtime.